prototypes/vue-reader/src/cite-services/components/Reader.vue

```diff
@@ -0,0 +1,298 @@
+<template>
+  <div class="root">
+    <header>
+      <div class="reader-nav">
+        <a href="/#/cts/">CTS API (Perseids)</a>
+        &bull;
+        <a href="/#/cite-services/">cite-services</a>
+        &bull;
+        <a href="/#/morphgnt/">MorphGNT API</a>
+      </div>
+      <h1><b>LORE</b>: Learner’s Online Reading Environment (cite-services)</h1>
+    </header>
+    <div class="grid-wrapper">
+      <div class="left">
+        <CtsUrn></CtsUrn>
+      </div>
+
+      <div class="main">
+        <template v-if="passage">
+
+          <pagination :prev="passageLink(query, passage.prev)" :next="passageLink(query, passage.next)" :title="passage.title"></pagination>
+
+          <div id="text" :class="'textSize-' + this.textSize">
+            <div v-for="node in passage.text.Nodes">
+              {{ node.text }}
+            </div>
+          </div>
+
+          <pagination :prev="passageLink(query, passage.prev)" :next="passageLink(query, passage.next)" :title="passage.title"></pagination>
+
+        </template>
+        <div v-else>
+          Nothing here yet.
+        </div>
+      </div>
+      <div class="right">
+        <text-formatting></text-formatting>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import fetch from 'universal-fetch';
+
+import Pagination from '@/components/Pagination';
+import TextFormatting from '@/components/TextFormatting';
+import CtsUrn from '@/cite-services/components/CtsUrn';
+
+export default {
+  name: 'reader',
+  created() {
+    this.fetchData();
+  },
+  mounted() {
+    window.addEventListener('keyup', this.handleKeyUp);
+  },
+  beforeDestroy() {
+    window.removeEventListener('keyup', this.handleKeyUp);
+  },
+  data() {
+    return {
+      query: null,
+    };
+  },
+  computed: mapGetters(['user', 'passage', 'textSize']),
+  watch: {
+    $route: 'fetchData',
+  },
+  methods: {
+    fetchData() {
+      this.asyncData({ query: this.$route.query }).then(({ query, passage }) => {
+        this.query = query;
+        this.$store.commit('setReader', { passage });
+      });
+    },
+    async asyncData({ query }) {
+      const urn = query.urn;
+      const url = `http://localhost:8080/million/texts/${urn}`;
```

Comment:
Is there going to be a public URL for this service or is local the only way for a while?